pkg/controller/clusters/status.go

```diff
@@ -253,6 +267,18 @@ func updateWorkersReadyCondition(cluster *anywherev1.Cluster, machineDeployments
 		return
 	}
 
+	for _, md := range machineDeployments {
```

Comment:
Add a comment that we are going through all the ASG machineDeployments now. Also, we could probably store these ones in another map rather than having to loop through again, but I guess that increases the storage.

Reply:
I will add a comment for now.